timesheet_multi_line_wizard_grid/static/src/js/grid_controller.js

```diff
@@ -0,0 +1,46 @@
+odoo.define("timesheet_multi_line_wizard_grid.GridController", function (require) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['odoo' is not defined.](https://app.codacy.com/manual/numigi/odoo-timesheet-addons/pullRequest?prid=4786394)